packages/firebase_data_connect/firebase_data_connect/lib/src/generated/connector_service.pbenum.dart

```diff
@@ -1,7 +1,3 @@
-// Copyright 2024, the Chromium project authors.  Please see the AUTHORS file
```

Comment:
I appreciate this is quite annoying, but all the generated src code has lost their license headers.

Reply:
It looks like the melos scripts aren't actually adding the headers. I'll do it manually

Reply:
Updated

Reply:
just noticed this in the melos script: https://github.com/firebase/flutterfire/blob/main/melos.yaml#L285. Not sure if it needs to be there but this is why you're probably not seeing it added.

Reply:
Ah, makes sense